src/lib/y2storage/proposal/autoinst_vg_planner.rb

```diff
@@ -0,0 +1,249 @@
+      # @param section [AutoinstProfile::PartitionSection] AutoYaST specification
+      def add_lv_reuse(planned_lv, planned_vg, section)
+        lv_to_reuse, vg = find_lv_to_reuse(planned_vg.volume_group_name, section)
+        return unless lv_to_reuse
+        planned_lv.logical_volume_name ||= lv_to_reuse.lv_name
+        planned_lv.filesystem_type ||= lv_to_reuse.filesystem_type
+        add_device_reuse(planned_lv, lv_to_reuse, section)
+        add_device_reuse(planned_lv.thin_pool, vg, section) if planned_lv.thin_pool
+      end
+
+      # Set 'reusing' attributes for a volume group
+      #
+      # This method modifies the first argument setting the values related to
+      # reusing a volume group (reuse and format).
+      #
+      # @param planned_vg [Planned::LvmVg] Planned volume group
+      # @param drive      [AutoinstProfile::DriveSection] drive section describing
+      #   the volume group
+      def add_vg_reuse(planned_vg, drive)
+        planned_vg.make_space_policy = drive.keep_unknown_lv ? :keep : :remove
+
+        return unless planned_vg.make_space_policy == :keep || planned_vg.all_lvs.any?(&:reuse?)
+        vg_to_reuse = find_vg_to_reuse(planned_vg, drive)
+        planned_vg.reuse_name = vg_to_reuse.vg_name if vg_to_reuse
+      end
+
+      # Set 'reusing' attributes for a thin pool logical volume
+      #
+      # This method modifies the argument setting the values related to reusing
+      # a thin logical volume (reuse_name). A thin pool will be planned to be
+      # reused if any of its logical volumes will be reused.
+      #
+      # @param planned_lv [Planned::LvmLv] Thin logical volume
+      def add_thin_pool_lv_reuse(planned_lv, _drive)
+        return unless planned_lv.thin_lvs.any?(&:reuse?)
+        lv_to_reuse = devicegraph.lvm_lvs.find { |v| planned_lv.logical_volume_name == v.lv_name }
+        planned_lv.reuse_name = lv_to_reuse.name
+      end
+
+      # @param vg_name     [String]      Volume group name to search for the logical volume to reuse
+      # @param part_section   [AutoinstProfile::PartitionSection] LV specification from AutoYaST
+      def find_lv_to_reuse(vg_name, part_section)
+        parent = find_lv_parent(vg_name, part_section)
+        return if parent.nil?
+        device = find_lv_in_vg(parent, part_section)
+
+        case device
+        when Y2Storage::LvmLv
+          [device, parent]
+        when :missing_reuse_info
+          issues_list.add(:missing_reuse_info, part_section)
+          nil
+        else
+          issues_list.add(:missing_reusable_device, part_section)
+          nil
+        end
+      end
+
+      # Finds a logical volume within a volume group
+      #
+      # @param vg           [LvmVg] Volume group to search for the logical volume
+      # @param part_section [AutoinstProfile::PartitionSection] LV specification from AutoYaST
+      # @return [LvmLv,Symbol Logical volume or nil if it is not found or there is not
```

Comment:
This piece of yardoc looks weird. Not only wrongly formatted but also not explaining well the `Symbol` case.